### DISCUSSION TITLE
Are there desktop notifications?

### DISCUSSION THREAD

**Discussion by gaveitatry2** [violentmonkey/violentmonkey#1237] [Q&A]
I am using 2.12.11.

I have two laptops.  My Acer Aspire 5.  And my Acer Nitro 5.  The Acer Aspire is my old laptop and my Acer Nitro is my new laptop.  Around a month ago, I briefly glimpsed a desktop notification on my old laptop that I THINK was a Violentmonkey notification that there was a new version available.  And today, just now, I received a desktop notification on my new laptop that I THINK belonged to Violentmonkey, but I didn't get a chance to read it because my new laptop was on the other side of the room and I was lying in bed.

I don't really understand.  In Settings, I have this box unchecked "Notify script updates".  Why am I getting desktop notifications?  I never ever had a browser extension give me desktop notifications before and I don't like it.

**Comment by tophf** [comment]
The only explanation is that those were network errors which currently are displayed regardless of the option. In the next release Violentmonkey won't display them if the option to notify updates is disabled. You can install the latest BETA v2.12.11.2.

**Reply by gaveitatry2** [reply]
I'm done with BETA. The only two things that I cared about was being able to use word-wrap again and I needed the 2.12.11 version so my scripts would work and load faster.  But one thing that I noticed is that the "Notify script updates" is for the userscript updates not ViolentMonkey extension updates.  So why am I getting desktop notifications?  I don't think that any of my scripts updated recently.  I can look at the "date" on the userscripts, and they haven't been updated in like 24 days.  There does NOT seem to be a way to disable the ViolentMonkey extension update desktop notifications (not script updates).  If there was a place to disable them, I haven't found it.  Where is it located?

**Reply by gaveitatry2** [reply]
![firefox_BVpkSPWFwE](https://user-images.githubusercontent.com/75463665/112602435-acba9880-8de1-11eb-9b1e-d13882465a7f.png)

You can see my standard Settings.  I can disable "Notify Script Updates" but I don't see an option to disable "ViolentMonkey Extention Updates".

**Reply by tophf** [reply]
Violentmonkey doesn't check its own updates so these notifications aren't displayed by Violentmonkey.
